src/StickyTree.jsx

```diff
@@ -751,6 +751,10 @@ export default class StickyTree extends React.PureComponent {
             pos = this.backwardSearch(scrollTop, currNodePos);
         }
 
+        if (scrollTop === this.state.scrollTop) {
```

Comment:
I think you want to do this check before the pos forward / backwards search above. Would be good to add a comment to the effect "ensure that we have actually changed scrollTop before we re-render the nodes"

Reply:
Let me know if you can push this, otherwise I take a look.

Reply:
Hi Mark, Thanks for pointing this. I have updated the code as suggested. Please review/merge and publish with a new version.